packages/arb-token-bridge-ui/src/ui-driver/UiDriver.ts

```diff
@@ -0,0 +1,50 @@
+export type UiDriverContext = {
+  isDepositMode: boolean
+  isSmartContractWallet: boolean
+}
+
+export type UiDriverStep =
+  | { type: 'start' } //
```

Comment:
added the comment for nicer code formatting. it will be removed in the next pr anyway